man/man8/zed.8.in

```diff
@@ -66,6 +67,12 @@ Lock all current and future pages in the virtual memory address space.
 This may help the daemon remain responsive when the system is under heavy
 memory pressure.
 .TP
+.BI \-I
+Make daemon to Idle if the kernel modules are not loaded, or if they
+are unloaded.
+The daemon will start, or resume, processing events when able.
+Unloading detection is not supported on all platforms.
+.TP
```

Comment:
How about something like this:

"Request that the daemon idle rather than exit when the kernel modules are not loaded.  Processing of events will start, or resume, when the kernel modules are (re)loaded.  Under Linux the kernel modules cannot be unloaded while the daemon is running."